sdk/extensions/Microsoft.Extensions.Azure/CHANGELOG.md

```diff
@@ -4,7 +4,7 @@
 
 ### Features Added
 
-- Added support for constructing a `DefaultAzureCredential` or `ManagedIdentityCredential` from config by setting the `managedIdentityObjectId` key.
+- Added support for constructing a `ManagedIdentityCredential` from config by setting the `managedIdentityObjectId` key.
```

Comment:
Mostly just curious here -- it looks like you changed the changelog -- was it incorrect before?  Did we add support for both or remove one?

Reply:
We added support for objectId to ManagedIdentityCredential, but not DAC. Your comment about the test logic reminded me that what I originally said here was incorect for DAC.